### ISSUE TITLE
Enforce passing specs

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#157]
# Summary
Our pipeline runs a test suite in gitlab ci. Currently, we are able to merge code into main with failing specs. 

After #23 is fixed with a green pipeline, please update .gitlab-ci.yml to enforce passing specs when merging into main. 

# Acceptance Criteria

* [x] If a branch has failing specs, it should not be allowed to get merged into main

@summer-cook er If you need help with this task, consult the docs for allow_failure flags or have @aprilrieger show you how to do this.

**Comment by orangewolf** [comment]
April January 2022
---
Enabled!

Ref: https://docs.gitlab.com/ee/user/project/merge_requests/merge_when_pipeline_succeeds.html#only-allow-merge-requests-to-be-merged-if-the-pipeline-succeeds

Video how-to: https://share.getcloudapp.com/L1uWQoWj

**Comment by orangewolf** [comment]
Diem January 2022
---
**This passes Notch8 QA.** See attached screenshot.
<details><summary>Following April's how-to video, I can see that "Pipelines must succeed" has been enabled on this project</summary>![General___Settings___notch8_Louisville-hyku___GitLab___2022-01-21_at_11.12.25_AM](/uploads/df4ba74e7de1583013c486d1bb1c45fc/General___Settings___notch8_Louisville-hyku___GitLab___2022-01-21_at_11.12.25_AM.jpg)</details>
